### PR TITLE
docs: enable dumi codesplitting jsStrategy

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

试试提取效果。
https://d.umijs.org/config#codesplitting

@PeachScript 默认值是 null，属于哪个策略？

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     --      |
| 🇨🇳 Chinese |    --       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 63f7eb5</samp>

Update Next.js example with code splitting configuration. Add `jsStrategy: 'granularChunks'` to `.dumirc.ts` to optimize loading and caching of JavaScript bundles.

### <samp>🤖 Generated by Copilot at 63f7eb5</samp>

*  Add code splitting configuration for Next.js framework ([link](https://github.com/ant-design/ant-design/pull/45214/files?diff=unified&w=0#diff-d6c33c311798bd8d66d8afcbff667cd5a6d78d8ac1813f70fecf4d4074420daaR25-R27))

### CODE REVIEW

File: .dumirc.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ export default defineConfig({
     atomDirs: [{ type: 'component', dir: 'components' }],
     codeBlockMode: 'passive',
   },
+  codeSplitting: {
+    jsStrategy: 'granularChunks',
```
Comment (kiner-tang) [inline .dumirc.ts:26]: 这个有什么作用呀

Comment (kiner-tang) [reply .dumirc.ts:26]: 提供的链接打不开：
image

**Comment by PeachScript** [comment]
默认值 `null` 代表按 async chunk 拆分，不做公共 chunk 提取，加了策略会把部分三方包提成独立的 chunk，好处是公共 chunk 名不变的时候浏览器缓存可以复用

File: .dumirc.ts (hunk 1 above)
Comment (PeachScript) [reply .dumirc.ts:26]: 这两天陆续有收到 dumi 官网打不开的反馈，可能 Vercel 的 IP 被 ban 了，要挂🪜？

**Comment by MadCcc** [comment]
之前试过，开了之后变化不大

**Comment by PeachScript** [comment]
> 之前试过，开了之后变化不大

preview 里 framework 的 chunk 没生成出来，应该是哪里工作不正常了，还没来得及看

**Comment by afc163** [comment]
> 所以 Umi 的这个配置值其实是无效的，这个 PR 可以关闭了🤦‍♂️

@sorrycc

**Pull request closed without merging**